src/ace.F90

```diff
@@ -740,14 +732,15 @@ subroutine read_reactions(nuc)
     ! Store elastic scattering cross-section on reaction one -- note that the
     ! sigma array is not allocated or stored for elastic scattering since it is
     ! already stored in nuc % elastic
-    rxn => nuc % reactions(1)
-    rxn % MT            = 2
-    rxn % Q_value       = ZERO
-    rxn % multiplicity  = 1
-    rxn % threshold     = 1
-    rxn % scatter_in_cm = .true.
-    rxn % has_angle_dist = .false.
-    rxn % has_energy_dist = .false.
+    associate (rxn => nuc % reactions(1))
+      rxn % MT            = 2
```

Comment:
In this case, the changes resulted from an extra level of indentation, so I didn't really touch those lines, per se. Any new code I'm writing I tend to use non-whitespace-separated % operator, as you've noticed.